.drone.yml

```diff
@@ -1558,6 +1558,450 @@ volumes:
   - name: dockersock
     temp: {}
 
+---
+kind: pipeline
+type: exec
+name: build-darwin-amd64
+
+concurrency:
+  limit: 1
+
+platform:
+  os: darwin
+  arch: amd64
+
+# trigger:
+#   event:
+#     - tag
+#   ref:
+#     include:
+#       - refs/tags/v*
```

Comment:
Remove comments.

Reply:
This is already done - bit of a race condition where I marked this ready for review, but then had to go back and comment some stuff for additional testing.